### PR TITLE
update: dockerfile from node:16.17.1-buster

### PR DESCRIPTION
**Pull request by askonev**
[node:16.17.1-buster](https://hub.docker.com/layers/library/node/16.17.1-buster/images/sha256-674750127bbf45f52660ada71ed1f1491d15e94c16583bff6df0df2489481049?context=explore)

Running tests on the [Karma](https://karma-runner.github.io/latest/index.html) and [Jasmine](https://jasmine.github.io/) frameworks

* building a Dockerfile `docker-build/docker-actions/Dockerfile` in which we install the browser `chrome:latest`
* running unit tests

### CODE REVIEW

File: .github/workflows/jasmine.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    branches:
+      - master
+
+jobs:
+  units:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
```
Comment (VitaliySerov) [inline .github/workflows/jasmine.yml:17]: v4 is the latest

File: docker-build/docker-actions/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
+FROM node:16.17.1-buster
```
Comment (VitaliySerov) [inline docker-build/docker-actions/Dockerfile:1]: Maybe call the dir for it more clear - not `docker-actions` but maybe something `unit-test-runner`

Comment (VitaliySerov) [reply docker-build/docker-actions/Dockerfile:1]: Please make sure that this docker is part of dependabot and will be update at same time as default dockerfile

**Review by VitaliySerov — CHANGES_REQUESTED** [review]
Looks very good, a great job

Some minor changes 

Also please check the action log:

https://github.com/palladium-tms/palladium-view/actions/runs/8572163899/job/23494065553?pr=430

It shows a lot of errors:
```
ERROR: 'NG0304: 'mat-error' is not a known element:
1. If 'mat-error' is an Angular component, then verify that it is part of this module.
2. If 'mat-error' is a Web Component then add 'CUSTOM_ELEMENTS_SCHEMA' to the '@NgModule.schemas' of this component to suppress this message.'
```

but test seems working, better to fix those

**Comment by askonev** [comment]
> Looks very good, a great job
> 
> Some minor changes
> 
> Also please check the action log:
> 
> https://github.com/palladium-tms/palladium-view/actions/runs/8572163899/job/23494065553?pr=430
> 
> It shows a lot of errors:
> 
> ```
> ERROR: 'NG0304: 'mat-error' is not a known element:
> 1. If 'mat-error' is an Angular component, then verify that it is part of this module.
> 2. If 'mat-error' is a Web Component then add 'CUSTOM_ELEMENTS_SCHEMA' to the '@NgModule.schemas' of this component to suppress this message.'
> ```
> 
> but test seems working, better to fix those

There was an error at startup
```log
(node:19) [DEP0148] DeprecationWarning: Use of deprecated folder mapping "./" in the "exports" field module resolution of the package at /palladium-view/node_modules/postcss/package.json.
Update this package.json to use a subpath pattern like "./*".
```
means that the postcss package is using an outdated way of specifying module exports in its package.json. This is a warning about future compatibility and does not usually affect the functionality of the package immediately but indicates that the package might need an update or configuration change in the future.

File: unit-test-runner/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    && rm -rf /var/lib/apt/lists/*
+
+RUN mkdir /palladium-view
+COPY . /palladium-view
+WORKDIR /palladium-view
+
+RUN npm install
+
+ENTRYPOINT ["npm"]
+CMD ["run", "ng", "test"]
```
Comment (VitaliySerov) [inline unit-test-runner/Dockerfile:19]: Question - do we really need a separate dockerfile to run unit tests?
Can we do it without docker - just install all deps inside GitHub Actions and run some comments?

I think that way we will be little bit more flexible in future - if we need we can create a dockerfile later
But having this command as native commands will allow us to run unit-tests easily on dev machine while development some feature without a need to constant rebuilding the dockerfile to rerun all the tests

In my option we need Dockerfile only for some production check, but unit-tests can be done on machined/GitHub Action directly

**Review by VitaliySerov — APPROVED** [review]

**Review by VitaliySerov — APPROVED** [review]
Nice job

**Pull request merged**